core/cap-0046-01.md

```diff
@@ -139,20 +139,22 @@ Multiple WASM VMs in the same host environment can refer to the same host object
 
```

Comment:
> Multiple WASM VMs in the same host environment can refer to the same host objects

So if A->B and B returned back an object handle O, then A will get an invalid object handle since B already finished and its frame has been rolled back?

Reply:
fixed